db/schema.rb

```diff
@@ -10,14 +10,42 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```

Comment:
Looks like there was a merge conflict in this file. The easiest fix, since you weren't touching this file, is probably to `git fetch && git checkout master db/schema.rb`, then commit that.

Reply:
That should update your master branch (git fetch) and then replace your branch's db/schema.rb with master's.